tests/test_sparql_store.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict
+import pytest
+
+from store.sparql import SparqlStore
+
+
+def test_run_sparql():
```

Comment:
you'll need a few more tests here:

- 1.) "test_sparql_store_setup_default" - test that when you instantiate a SparqlStore with no env var the default url is set
- 2.) "test_sparql_store_setup_custom" -  test that you can specify a different url via an env var and the SparqlStore will use that instead.
- 3.) "test_sparql_store_run_sparql" needs to test the _run_sparql_ method and confirm its returning a SPARQLWrapper.QueryResult.
- 4.) "test_sparql_store_get_datasets" needs to test the _get_datasets_ method and confirm its returning a dict.